tests/unit/test_models.py

```diff
@@ -377,13 +389,27 @@ class GrammarValidation(pydantic.BaseModel):
     err = raised.value.errors()
     assert len(err) == 1
     assert err[0]["loc"] == ("x",)
-    assert err[0]["type"] == "value_error"
-    assert err[0]["msg"] == f"Value error, value must be a str: {value!r}"
+    assert err[0]["type"] == "string_type"
+    assert err[0]["msg"] == "Input should be a valid string"
```

Comment:
This change uses the standard pydantic form when the type isn't a dictionary. (The dict type was moved to `test_grammar_errors` below)